google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Heartbeat.java

```diff
@@ -29,22 +32,30 @@ public abstract class Heartbeat implements ChangeStreamRecord, Serializable {
   private static final long serialVersionUID = 7316215828353608504L;
 
   private static Heartbeat create(
-      ChangeStreamContinuationToken changeStreamContinuationToken, Instant estimatedLowWatermark) {
-    return new AutoValue_Heartbeat(changeStreamContinuationToken, estimatedLowWatermark);
+      ChangeStreamContinuationToken changeStreamContinuationToken,
+      java.time.Instant estimatedLowWatermark) {
+    return new AutoValue_Heartbeat(
+        changeStreamContinuationToken, toThreetenInstant(estimatedLowWatermark));
   }
 
   /** Wraps the protobuf {@link ReadChangeStreamResponse.Heartbeat}. */
   static Heartbeat fromProto(@Nonnull ReadChangeStreamResponse.Heartbeat heartbeat) {
     return create(
         ChangeStreamContinuationToken.fromProto(heartbeat.getContinuationToken()),
-        Instant.ofEpochSecond(
+        java.time.Instant.ofEpochSecond(
             heartbeat.getEstimatedLowWatermark().getSeconds(),
             heartbeat.getEstimatedLowWatermark().getNanos()));
   }
 
   @InternalApi("Intended for use by the BigtableIO in apache/beam only.")
   public abstract ChangeStreamContinuationToken getChangeStreamContinuationToken();
 
+  /** This method is obsolete. Use {@link #getEstimatedLowWatermarkInstant()} instead. */
+  @ObsoleteApi("Use getEstimatedLowWatermarkInstant() instead")
+  public abstract org.threeten.bp.Instant getEstimatedLowWatermark();
```

Comment:
Same question as in ChangeStreamMutation. Wouldn't it be better to store this as java.time.Instant?